docs/src/components/ColorSwatch/ColorSwatch.component.tsx

```diff
@@ -0,0 +1,104 @@
+// REACT
+import * as React from 'react';
+// VENDOR
+import styled, { css } from 'styled-components';
+import classNames from 'classnames';
+import { Pill } from '../Pill'
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+// THEME
+import { colors, sizes } from '../../../../src/theme';
+
+
+// INTERFACES
+interface ColorSwatchProps {
+  className?: string;
+  backgroundColor: string;
+  contrastOnBlack: number;
+  contrastOnF1F1F1: string;
+  contrastOnFAFAFA: string;
+  contrastOnWhite: number;
+  copyChoice: string;
+  name: string;
+  size: 'large' | 'small';
+  textColor: string;
+}
+
+interface ColorSwatchStylesGroup {
+  [key: string]: FlattenSimpleInterpolation;
+}
+
+
+// STYLES
+const ColorSwatchSizeStyles: ColorSwatchStylesGroup = {
+  large: css`
+    .ColorSwatchPillContainer {
+      padding-top: 200px
+    }
+  `,
+  small: css`
+    .ColorSwatchPillContainer {
+      padding-top: 50px
+    }
+  `,
+}
+
+const StyledColorSwatch = styled.div`
+  background: ${props => props.backgroundColor };
+  color: ${props => props.textColor };
+  padding: 24px 16px 16px 16px;
+
+  .ColorSwatchName {
+    margin: 0;
+    font-weight: 600;
+  }
+
+  .ColorSwatchBackgroundColor {
+    margin: 0;
+  }
+
+  ${({ size = 'large' }: ColorSwatchProps) => ColorSwatchSizeStyles[size]}
+`;
+
+
+// JSX MARKUP
+export const ColorSwatch = ({
+  className,
+  backgroundColor,
```

Comment:
In my opinion, `ColorSwatch` should take a `color` or something similarly named (e.g. `swatchColor`, `themeColor` etc) as a prop to go along with the swatch `name`, which is then used as the background color, instead of specifying a backgroundColor explicitly. 

That way the way in which the swatch decides to present the swatch color is decoupled from its interface.